environment-pipelines/buildspec-trigger.yml

```diff
@@ -0,0 +1,50 @@
+version: 0.2
+
+env:
+  parameter-store:
+    SLACK_TOKEN: /codebuild/slack_oauth_token
+
+phases:
+  install:
+    commands:
+      - export PATH="${CODEBUILD_SRC_DIR}/build-tools/bin:$PATH"
+      - export PYTHONPATH="${CODEBUILD_SRC_DIR}/build-tools"
+      - echo -e "\nAssume triggered account role to trigger ${TRIGGERED_PIPELINE_NAME} pipeline"
+
+      - assumed_role=$(aws sts assume-role --role-arn "${TRIGGERED_ACCOUNT_ROLE_ARN}" --role-session-name "trigger-prod-pipeline-$(date +%s)")
+
+      - PROD_AWS_ACCESS_KEY_ID=$(echo $assumed_role | jq -r .Credentials.AccessKeyId)
+      - PROD_AWS_SECRET_ACCESS_KEY=$(echo $assumed_role | jq -r .Credentials.SecretAccessKey)
+      - PROD_AWS_SESSION_TOKEN=$(echo $assumed_role | jq -r .Credentials.SessionToken)
+
+      - export PROFILE_NAME="${TRIGGERED_PIPELINE_AWS_PROFILE}"
+      # Populate the ~/.aws/credentials file..
+      - aws configure set aws_access_key_id "${PROD_AWS_ACCESS_KEY_ID}" --profile "${PROFILE_NAME}"
+      - aws configure set aws_secret_access_key "${PROD_AWS_SECRET_ACCESS_KEY}" --profile  "${PROFILE_NAME}"
+      - aws configure set aws_session_token "${PROD_AWS_SESSION_TOKEN}" --profile  "${PROFILE_NAME}"
+      # Populate the ~/.aws/config file..
+      - aws configure set region "eu-west-2" --profile  "${PROFILE_NAME}"
+      - aws configure set output "json" --profile  "${PROFILE_NAME}"
+
+  build:
+    commands:
+      - set -e
+      - message="Triggering ${TRIGGERED_PIPELINE_NAME} pipeline"
```

Comment:
Maybe uppercase "message" for consistency.

Reply:
ok